model/src/iosMain/kotlin/io/github/droidkaigi/feeder/Locale.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.droidkaigi.feeder
+
+import platform.Foundation.NSLocale
+import platform.Foundation.countryCode
+import platform.Foundation.currentLocale
+
+actual fun getDefaultLocale(): Locale =
+    if (NSLocale.currentLocale.countryCode == "JP") {
```

Comment:
There is no need to fix it, but if you want to start a new line here, it may be better to use block according to the style guide.
https://developer.android.com/kotlin/style-guide#expression_functions

Reply:
Thanks for your advice.
I thought it is normal according to the implementation for Android, but it seems like that this was only because the code didn't fit on a single line.